### ISSUE TITLE
Groups View - User created groups

### ISSUE THREAD

**Issue by yuisotozaki** [bcgov/ckan-ui#133]
All test participants showed interest in being able to create and share their own group of datasets.
Add support for user created groups.
This requires business input first.

**Comment by SteveChapmanBCDX** [comment]
Good idea.  Definitely not something that we will include in MVP, but good idea down the road.

**Comment by pumapants** [comment]
How Group security is managed will need to be reviewed as this is only done by sysadmins and realm admins at this time. Security would probably then need to move out of our Realm and into the cat itself.

**Comment by TerryLanktree** [comment]
This sounds like something we did in Distribution Service a long time ago.  If a user is logged in, they could save an order (group of layers) to retrieve at a later date.  This was useful for clients who downloaded the same x number of datasets regularly.  We limited to 3 scenarios that could be saved.  It took sup space in the database, but the clients who used it, loved it.  We would simply need a way to then "share" the list.  I see this would be great for business areas, layers you need when doing a dock permit, or some such.

**Comment by yuisotozaki** [comment]
The users we interviewed were interested in being able to accomplish these things with the group concept:
1. I want to put a bunch of datasets together so that I can share with my colleagues/store the link to the group in my reference doc. I want people from my organization to be able to see this group and maintain it long term.
2. I want to keep an eye on a bunch of datasets for updates - I want to be notified about updates to any of the datasets in the group I created. Email notification is fine.
3. I want to create a group of datasets under a 'theme' so that I can share this as a way for the general public to find relevant set of datasets (same purpose as the covid-19 group).

**Comment by sheilamv** [comment]
https://dpdd.atlassian.net/browse/DDS-462
